templates/whatson/whats_on_page.html

```diff
@@ -87,4 +87,77 @@ <h2 class="tna-heading tna-heading--l event-card-listing__heading">All events</h
       </ul>
     </div>
   </div>
+
+  <div class="section section--pink">
+    <div class="tna-container">
+        <div class="tna-column tna-column--full tna-!--margin-top-l">
+            <h2 class="tna-heading related-content__heading">For professionals</h2>
```

Comment:
I think we might also need a `tna-heading--m`, `tna-heading--l` or `tna-heading--xl` class

Reply:
Hey @ahosgood I've updated with those headings now :) are you happy to sign this off so we can get it merged in, ready for the BE to be updated?

Reply:
Sorry, of course 👍🏼